app/models/iiif_record.rb

```diff
@@ -0,0 +1,51 @@
+class IIIFRecord
+  def initialize(solr_doc)
+    @solr_doc = solr_doc
+  end
+
+  def to_json
+    { label: label, description: description, metadata: metadata }
+  end
+
+  def authors
+    authors_json = (@solr_doc['author_roles_1display'] || []).first
+    return {} unless authors_json
+    JSON.parse(authors_json)
+  end
```

Comment:
I would probably use the `author_display` which is for the the primary author (Marc 100/110/111) and its accompanying localized script version. Not all records have a primary author. If we have a preference for localized scripts, it always appears after the romanized version. 

How interested are we in creator roles (and including more than one creator)? I don't think the `author_roles_1display` field is reliable. The roles included there are supposed to conform to various citation standards, but the `related_name_json_1display` actually has a more accurate classification of roles. 

If there is a more specific role than "author" encoded for the primary author, it is stored in the `marc_relator_display` field. Gotta love Marc.